requirements.pip

```diff
@@ -1,14 +1,16 @@
 --extra-index-url https://staticfiles.krxd.net/foss/pypi/
 # direct dependencies
 sh==1.11
-virtualenv-tools==1.0
+# note that this virtualenv-tools is our fork, not the public one, the source is
+# here: https://github.com/krux/virtualenv-tools
+virtualenv-tools==1.0.1
```

Comment:
Yes. However, the chances of upstream merging *anything* are astronomically small; they won't even answer why they won't merge anything; there are several open [PRs](https://github.com/fireteam/virtualenv-tools/pulls) with similar changes, all without any maintainer comments.